From e68ce857cd6a5a630060b3aad4b10a3555b18d21 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Mon, 12 Apr 2010 17:22:32 +0200
Subject: [PATCH 29/33] cleanup 10-usb-music-players.fdi, sort by vendor ID's

Cleaned up 10-usb-music-players.fdi. Sort entries by
vendor ID's.
---
 .../10freedesktop/10-usb-music-players.fdi         |  150 ++++++++++----------
 1 files changed, 75 insertions(+), 75 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index 3a3d352..49bf7c3 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -576,6 +576,17 @@
           </match>
 	</match>
 
+        <!-- TrekStor -->
+        <match key="@storage.originating_device:usb.vendor_id" int="0x0738">
+          <!-- i.Beat cebrax fx 1GB -->
+          <match key="@storage.originating_device:usb.product_id" int="0x1738">
+            <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+            <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+            <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+            <append key="portable_audio_player.input_formats" type="strlist">audio/x-wav</append>
+          </match>
+        </match>
+
         <!-- Olympus -->
         <match key="@storage.originating_device:usb.vendor_id" int="0x07b4">
           <!-- DS-2 voice recorder -->
@@ -585,6 +596,43 @@
           </match>
         </match>
 
+	<!-- Palm -->
+	<match key="@storage.originating_device:usb.vendor_id" int="0x0830">
+          <!-- Palm Pre -->
+          <match key="@storage.originating_device:usb.product_id" int="0x8004">
+             <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/3gpp</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/mp4</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+             <append key="portable_audio_player.audio_folders" type="strlist">Music/</append>
+          </match>
+        </match>
+
+        <!-- High Tech Computer Corp. -->
+        <match key="@storage.originating_device:usb.vendor_id" int="0xbb4">
+          <!-- Android Phone, HTC Diamond -->
+          <match key="@storage.originating_device:usb.product_id" int_outof="0xc01;0xc02;0xc13">
+            <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/3gpp</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/mp4</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/vnd.rn-realaudio</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-midi</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+             <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
+             <append key="portable_audio_player.audio_folders" type="strlist">Music/</append>
+          </match>
+	  <!-- Windows Mobile PPC HTC Touch HD -->
+          <match key="@storage.originating_device:usb.product_id" int="0xc25">
+             <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+             <append key="portable_audio_player.audio_folders" type="strlist">My Music/</append>
+          </match>
+        </match>
+
 	<!-- Peak Digital Audio Player -->
         <match key="@storage.originating_device:usb.vendor_id" int="0xd7d">
           <match key="@storage.originating_device:usb.product_id" int="0x1651">
@@ -606,6 +654,14 @@
 	  </match>
 	</match>
 
+        <!-- ISCI, hama, no-name -->
+        <match key="@storage.physical_device:usb.vendor_id" int="0xdda">
+          <!-- IC1110 V1.0A, hama DMP-100 -->
+          <match key="@storage.physical_device:usb.product_id" int="0x302">
+            <merge key="portable_audio_player.access_method" type="string">storage</merge>
+          </match>
+        </match>
+
 	<!-- Cowon -->
 	<match key="@storage.originating_device:usb.vendor_id" int="0xe21">
 	  <!-- iAUDIO M3 -->
@@ -1007,17 +1063,6 @@
           </match>
         </match>
 
-        <!-- TrekStor -->
-        <match key="@storage.originating_device:usb.vendor_id" int="0x0738">
-          <!-- i.Beat cebrax fx 1GB -->
-          <match key="@storage.originating_device:usb.product_id" int="0x1738">
-            <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
-            <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
-            <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
-            <append key="portable_audio_player.input_formats" type="strlist">audio/x-wav</append>
-          </match>
-        </match>
-
 	<!-- Amazon -->
 	<match key="@storage.originating_device:usb.vendor_id" int="0x1949">
 	  <!-- Amazon Kindle -->
@@ -1027,6 +1072,25 @@
 	  </match>
 	</match>
 
+	<!-- Insignia -->
+	<match key="@storage.originating_device:usb.vendor_id" int="0x19ff">
+          <!-- Insignia NS-8V24 -->
+          <match key="@storage.originating_device:usb.product_id" int="0x0308">
+            <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+            <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
+            <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+            <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+            <append key="portable_audio_player.output_formats" type="strlist">video/mpeg4</append>
+            <append key="portable_audio_player.input_formats" type="strlist">audio/mpeg</append>
+            <append key="portable_audio_player.input_formats" type="strlist">audio/x-ms-wma</append>
+            <append key="portable_audio_player.audio_folders" type="strlist">music/</append>
+            <append key="portable_audio_player.audio_folders" type="strlist">video/</append>
+            <append key="portable_audio_player.audio_folders" type="strlist">record/</append>
+            <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
+            <append key="portable_audio_player.playlist_path" type="string">playlists/%File</append>
+          </match>
+        </match>
+
         <!-- Motorola -->
         <match key="@storage.originating_device:usb.vendor_id" int="0x22b8">
           <!-- Motorola iTunes enabled phone -->
@@ -1121,70 +1185,6 @@
           </match>
 	</match>
 
-        <!-- High Tech Computer Corp. -->
-        <match key="@storage.originating_device:usb.vendor_id" int="0xbb4">
-          <!-- Android Phone, HTC Diamond -->
-          <match key="@storage.originating_device:usb.product_id" int_outof="0xc01;0xc02;0xc13">
-            <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/3gpp</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/mp4</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/vnd.rn-realaudio</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/x-midi</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
-             <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
-             <append key="portable_audio_player.audio_folders" type="strlist">Music/</append>
-          </match>
-	  <!-- Windows Mobile PPC HTC Touch HD -->
-          <match key="@storage.originating_device:usb.product_id" int="0xc25">
-             <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
-             <append key="portable_audio_player.audio_folders" type="strlist">My Music/</append>
-          </match>
-        </match>
-
-        <!-- ISCI, hama, no-name -->
-        <match key="@storage.physical_device:usb.vendor_id" int="0xdda">
-          <!-- IC1110 V1.0A, hama DMP-100 -->
-          <match key="@storage.physical_device:usb.product_id" int="0x302">
-            <merge key="portable_audio_player.access_method" type="string">storage</merge>
-          </match>
-        </match>
-
-	<!-- Insignia -->
-	<match key="@storage.originating_device:usb.vendor_id" int="0x19ff">
-          <!-- Insignia NS-8V24 -->
-          <match key="@storage.originating_device:usb.product_id" int="0x0308">
-            <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
-            <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
-            <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
-            <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
-            <append key="portable_audio_player.output_formats" type="strlist">video/mpeg4</append>
-            <append key="portable_audio_player.input_formats" type="strlist">audio/mpeg</append>
-            <append key="portable_audio_player.input_formats" type="strlist">audio/x-ms-wma</append>
-            <append key="portable_audio_player.audio_folders" type="strlist">music/</append>
-            <append key="portable_audio_player.audio_folders" type="strlist">video/</append>
-            <append key="portable_audio_player.audio_folders" type="strlist">record/</append>
-            <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
-            <append key="portable_audio_player.playlist_path" type="string">playlists/%File</append>
-          </match>
-        </match>
-
-	<!-- Palm -->
-	<match key="@storage.originating_device:usb.vendor_id" int="0x0830">
-          <!-- Palm Pre -->
-          <match key="@storage.originating_device:usb.product_id" int="0x8004">
-             <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/3gpp</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/mp4</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
-             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
-             <append key="portable_audio_player.audio_folders" type="strlist">Music/</append>
-          </match>
-        </match>
-
       </match>
     </match>
 
-- 
1.7.1

